packages/web3-eth-contract/src/constants.ts

```diff
@@ -15,7 +15,7 @@ You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { AbiEventFragment } from 'web3-eth-abi';
+import { AbiEventFragment } from 'web3-types';
```

Comment:
Pls also include change log for these breaking changes of types moved to new package.